libraries/base/GHC/Event/PSQ.hs

```diff
@@ -28,7 +28,7 @@ module GHC.Event.PSQ
     , singleton
 
     -- * Insertion
-    , insert
+    , unsafeInsertNew
```

Comment:
If `insert` isn't used then presumably we can drop its binding.

Reply:
Also, it would be a good idea to update the comment on `unsafeInsertNew` since it is no longer internal.